eras/alonzo/test-suite/test/Test/Cardano/Ledger/Alonzo/Golden.hs

```diff
@@ -80,36 +80,38 @@ goldenUTxOEntryMinAda =
         calcMinUTxO
           ( AlonzoTxOut
               carlAddr
-              (valueFromList 1407406 [(pid1, smallestName, 1)])
+              (MaryValue 1407406 (multiAssetFromList [(pid1, smallestName, 1)]))
```

Comment:
I wonder if it wouldn't have been better to change the definition of valueFromList (just once), then we woudn't need to change any of the call sites?

There are more than a dozen in this file alone. Could we revert this file and just change the definition?

Reply:
Sound sensible. I think my idea was not so much to get rid of `valueFromList` as to expose `multiAssetFromList`, which is needed in places other the pattern you're pointing out, so I replaced one with the other. But there is no reason for which both can't exist.  Will make the change, thanks for the suggestion.